src/views/pages/ProductTransactions/index.vue

```diff
@@ -10,14 +10,37 @@
 
       <h2 class="error md-subheader">{{error}}</h2>
 
-      <md-table-row slot="md-table-row" slot-scope="{ item }">
+      <md-table-row slot="md-table-row" slot-scope="{ item }" md-selectable="single" md-auto-select>
         <md-table-cell md-label="Date" md-sort-by="datetime">{{item.datetime}}</md-table-cell>
         <md-table-cell md-label="From" md-sort-by="from">{{item.from}}</md-table-cell>
         <md-table-cell md-label="To" md-sort-by="to">{{item.to}}</md-table-cell>
         <md-table-cell :md-label="`Amount (${item.productUnits})`" md-sort-by="amountOfProduct">{{item.amountOfProduct}}</md-table-cell>
         <md-table-cell :md-label="`Rate (${item.productUnits}/${item.currency})`" md-sort-by="costPerUnit">{{item.costPerUnit}}</md-table-cell>
       </md-table-row>
     </md-table>
+
+    <div class="view-dialog-wrapper">
+      <md-dialog v-if="selectedRow" :md-active.sync="showViewDialog">
+        <md-dialog-title>Transaction Details</md-dialog-title>
+          <md-dialog-content>
+            <div class="md-subhead">
+              <md-icon>access_time</md-icon>
+              <span>Date: {{ selectedRow.datetime }}</span>
+            </div>
+            <h3><b>From</b></h3>
+            {{ selectedRow.from }}
+            <h3><b>To</b></h3>
+            {{ selectedRow.to }}
+            <h3><b>Amount ({{ selectedRow.productUnits }})</b></h3>            
+            {{ selectedRow.amountOfProduct }}
+            <h3><b>Rate ({{ selectedRow.productUnits }}/{{ selectedRow.currency }})</b></h3>            
```

Comment:
It's currency/productUnits

Reply:
Nice catch, I copied what the table column said :p

Reply:
Ah yea, can you fix that in this PR?